src/lib/Display.ts

```diff
@@ -0,0 +1,138 @@
+import {Spinner} from "@salesforce/sf-plugins-core";
+import {Ux} from "@salesforce/sf-plugins-core/lib/ux";
+
+export interface Display {
```

Comment:
As we inject this interface as a dependency everywhere, it should help us move entirely away from our display based events (see ScannerEvents.js).